qa/tasks/cephfs/kernel_mount.py

```diff
@@ -70,8 +45,8 @@ def mount(self, mount_path=None, mount_fs_name=None):
         if mount_path is None:
             mount_path = "/"
 
-        opts = 'name={id},secretfile={secret},norequire_active_mds'.format(id=self.client_id,
```

Comment:
Wouldn't it be better to use secretfile, by default, since you mentioned elsewhere that not passing secret is a risky option?

Reply:
That's talking about the password for sudo. That's unrelated to the cephx secret that the kernel needs to be able to authenticate.